app/javascript/orangelight/availability.es6

```diff
@@ -326,17 +322,8 @@ export default class AvailabilityUpdater {
     }
 
     update_request_button(holding_id, availability_info) {
```

Comment:
We update the request button to hide or show if it's on CDL. The removed code is not used.